unity-scene-reference/Assets/source/SceneReference.cs

```diff
@@ -6,6 +6,7 @@
 using UnityEditor;
 using UnityEditor.SceneManagement;
 using UnityEditor.VersionControl;
+using UnityEngine.SceneManagement;
```

Comment:
U already are using SceneManagement 2 lines above

Reply:
One is `UnityEditor` and the other is `UnityEngine`

Reply:
my bad xD